### ISSUE TITLE
Default avatar - User First letter of user's firstname & Lastname

### ISSUE THREAD

**Issue by Code4SocialGood** [Code4SocialGood/c4sg-web#993]
Currently we display a default avatar image
Please display the first letter of user's first name and last name

![image](https://user-images.githubusercontent.com/24813620/28844390-184902c2-76ca-11e7-9ce4-b19f7eb6df67.png)

**Comment by axlemax** [comment]
My changes will give any user (note: not the current user) without an avatar URL a colorful circle with their initials in it. If this should also happen with the CURRENT user, that will need to be a separate ticket.

**Comment by Code4SocialGood** [comment]
@axlemax When a new user registers and login, the user is considered as CURRENT user and doesn't get a colorful circle displayed on the header or displayed on My Settings page. Correct?

**Comment by axlemax** [comment]
Yes, exactly. The places where the current user avatar is seen is in the header or when editing a user, which is when the default avatar will be seen. Though when any other user / org views that user, they will see the circle. 

I thought of changing it there as well, but we specifically do the avatar for the header a lot differently: elsewhere we check if the user has an avatar URL or not (and display it if present), in the header we try to retrieve the image by URL, and display the default avatar when any error is encountered.
